internal/backend/oauth/svc.go

```diff
@@ -24,10 +24,10 @@ type server struct {
 
 var _ oauthv1connect.OAuthServiceHandler = (*server)(nil)
 
-func Init(mux *http.ServeMux, l *zap.Logger, oauth sdkservices.OAuth) {
+func Init(muxes *muxes.Muxes, l *zap.Logger, oauth sdkservices.OAuth) {
 	srv := server{logger: l, impl: oauth}
 	path, handler := oauthv1connect.NewOAuthServiceHandler(&srv)
-	mux.Handle(path, handler)
```

Comment:
are you sure oauth flow can be authenticated fully ?

Reply:
always authd now

Reply:
this will work on the next pr (#240 ), which I'll merge together, it's the other pr's concern.